weekly-updates/project-lead-meetings/10_14_18.md

```diff
@@ -89,13 +89,13 @@ _TO DO (for all of next semester)_:
 
 ## sledge
 
-_UPDATES_:
+_UPDATES_: The Good: It worked, ratings updated as expected, everyone got it running (with some refreshes) The Bad: Not enough judges on the top prizes, lots of very close scores, The Ugly: Table numbers didn't work
 
-_QUESTIONS_:
+_QUESTIONS_: What's a better system for choosing between close scores?
 
-_TO DO (for all of next semester)_:
+_TO DO (for all of next semester)_: General improvements, table system and better tie-breaking capabilities (maybe add more judges to top hacks)
```

Comment:
I was thinking of a dynamic thing where it'd push notify judges to break ties (like "we noticed these hacks judged by you are performing well, please pick a preference"). This might also not work in various cases, let us know.

Reply:
I'm going to start by giving top hacks more judges, if that's not good enough we can do something comparative. I'm hoping to run any ideas about scoring by my architects so we can be sure we have a robust solution.